src/clienthandler.rs

```diff
@@ -291,7 +292,19 @@ impl ClientHandler {
 									let relay_message = RelayMessage::new_event(sub_id.clone(), event.clone());
 									let serialized_message = relay_message.as_json();
 									match outgoing_send.send(serialized_message.into_bytes()) {
-										Ok(_) => {},
+										Ok(_) => {
+											let hash = hash_event(&event);
```

Comment:
Architecture-wise, I think moving at `OkEven` creation i.e after the event has been paid for its disk storage with our credential gateway and it is satisfying our server policy (`filter_events()`) and we have check the type is ephemeral.

So I think more for an integration at `OkEvent` creation in `kindprocessor`, we only need to commit the event id, as it’s already its own commitment.